plugins/modules/source_control/gitlab/gitlab_service.py

```diff
@@ -0,0 +1,467 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Raphaël Droz <raphael@droz.eu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: gitlab_service
+short_description: Creates, updates or deletes GitLab services (aka "integrations").
+version_added: '1.3.0'
+description:
+  - Setup or delete GitLab integration services.
+author:
+  - Raphaël Droz (@drzraf)
+requirements:
+  - python >= 2.6
+  - python-gitlab python module
+extends_documentation_fragment:
+- community.general.auth_basic
+
+options:
+  project:
+    description:
+      - The ID or path of the project (urlencoded or not)
+    required: true
+    type: str
+  active:
+    description:
+      - Not yet supported (U(https://gitlab.com/gitlab-org/gitlab-ce/issues/41113))
+    type: bool
+    default: yes
+    choices: [true]
+  service:
+    description:
+      - The service
+    required: true
+    type: str
+    choices:
+      - asana
+      - assembla
+      - bamboo
+      - bugzilla
+      - buildkite
+      - campfire
+      - custom-issue-tracker
+      - discord
+      - drone-ci
+      - emails-on-push
+      - external-wiki
+      - flowdock
+      - hangouts-chat
+      - hipchat
+      - irker
+      - jira
+      - mattermost
+      - mattermost-slash-commands
+      - microsoft-teams
+      - packagist
+      - pipelines-email
+      - pivotaltracker
+      - prometheus
+      - pushover
+      - redmine
+      - slack
+      - slack-slash-commands
+      - teamcity
+      - unify-circuit
+      - webex-teams
+      - youtrack
+  params:
+    description:
+      - The description of the service, see documentation U(https://docs.gitlab.com/ee/api/services.html)
+    type: dict
+  events:
+    description:
+      - The events that trigger the service (required if C(state=present))
+    choices: ["push", "issues", "confidential_issues", "merge_requests", "tag_push", "note", "confidential_note", "job", "pipeline", "wiki_page"]
+    default: ["push", "issues", "confidential_issues", "merge_requests", "tag_push", "note", "confidential_note", "job", "pipeline", "wiki_page"]
+    type: list
+  state:
+    description:
+      - Create or delete a service.
+      - Possible values are present and absent.
+    default: "present"
+    choices: ["present", "absent"]
+    type: str
+'''
+
+EXAMPLES = '''
+# Setup email on push for this project
+- name: emails me on push
+  gitlab_service:
```

Comment:
```suggestion
  community.general.gitlab_service:
```
Always use FQCN in examples.